server/komodo-rest/src/main/java/org/komodo/servicecatalog/datasources/ODataV4Definition.java

```diff
@@ -29,7 +30,7 @@
 
     @Override
     public String getType() {
-        return "odata4";
+        return "webservice";
```

Comment:
See Amazon class comment. Is this getType() more generic now?

Reply:
same as above.